Makefile

```diff
@@ -39,40 +30,12 @@ AWS_AMI_IMAGE=	nanos-$(TARGET)
 
 all: image
 
-include rules.mk
-
-.PHONY: image release contgen mkfs boot stage3 target stage distclean
-
-image: mkfs boot stage3 target
-	@ echo "MKFS	$@"
-	@ $(MKDIR) $(dir $(IMAGE))
-	$(Q) $(MKFS) $(TARGET_ROOT_OPT) -b $(BOOTIMG) $(IMAGE) <test/runtime/$(TARGET).manifest
-
-release: mkfs boot stage3
-	$(Q) $(RM) -r release
-	$(Q) $(MKDIR) release
-	$(CP) $(MKFS) release
-	$(CP) $(BOOTIMG) release
-	$(CP) $(STAGE3) release
-	cd release && $(TAR) -czvf nanos-release-$(REL_OS)-${version}.tar.gz *
-
-contgen:
-	$(Q) $(MAKE) -C $@
+.PHONY: image release target distclean
```

Comment:
The `release` and `target` Makefile targets don't do anything now in the top-level Makefile, so either they should be removed from the above line, or there should be a redirection to the appropriate platform directory, otherwise when running e.g. `make release` one gets the misleading message `make: Nothing to be done for 'release'`.